internal/commands/build/build.go

```diff
@@ -22,24 +23,27 @@ import (
 	"github.com/docker/buildx/util/progress"
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
+	"github.com/docker/cnab-to-oci/remotes"
 	"github.com/docker/distribution/reference"
 	"github.com/moby/buildkit/client"
 	"github.com/moby/buildkit/session"
 	"github.com/moby/buildkit/session/auth/authprovider"
 	"github.com/moby/buildkit/util/appcontext"
+	errors2 "github.com/pkg/errors"
```

Comment:
I know part of it as already been discussed, but I think we should avoid import names as `errors2`.
`errors` is used at only one place in this file:
https://github.com/docker/app/blob/1320c6034ba8391acd4c81618dae171d14a2187d/internal/commands/build/build.go#L254
And can be replaced by `fmt`

```go
return fmt.Errorf("'build' require docker engine 19.03 or later")
```

Reply:
Make sense.
I don't see any benefit in package prefixing used by Golang for imported functions and types. To me this is more noise than anything usefull.